diff -u src/tasks/src/cmatrix_base/cmatrix.c src/tasks/src/cmatrix/cmatrix.c
--- src/tasks/src/cmatrix_base/cmatrix.c	2026-04-06 19:15:22.394806534 +0200
+++ src/tasks/src/cmatrix/cmatrix.c	2026-04-06 19:14:06.777201469 +0200
@@ -847,7 +847,9 @@
                             wchar_t char_array[2];
                             char_array[0] = matrix[i][j].val;
                             char_array[1] = 0;
-                            addwstr(char_array);
+
+                            // TODO: Implement wide char support
+                            addstr((const char*)char_array); // addwstr(char_array);
                         }
                         if (bold == 2 ||
                             (bold == 1 && matrix[i][j].val % 2 == 0)) {
                             
diff -u src/tasks/src/cmatrix_base/config.sub src/tasks/src/cmatrix/config.sub
--- src/tasks/src/cmatrix_base/config.sub	2026-04-06 19:17:14.953053687 +0200
+++ src/tasks/src/cmatrix/config.sub	2026-04-06 18:40:52.822900343 +0200
@@ -2016,6 +2016,7 @@
 	| hcos* \
 	| hiux* \
 	| hms* \
+	| horizonos* \
 	| hpux* \
 	| ieee* \
 	| interix* \
